GPU/GPUCommon.cpp

```diff
@@ -1618,6 +1632,14 @@ void GPUCommon::Execute_Prim(u32 op, u32 diff) {
 			gstate.cmdmem[GE_CMD_TEXSCALEV] = data;
 			gstate_c.uv.vScale = getFloat24(data);
 			break;
+		case GE_CMD_TEXOFFSETU:
```

Comment:
Tomb Raider use GE_CMD_TEXOFFSETU and GE_CMD_TEXOFFSETV frequently, but not quite useful.:grimacing:

Reply:
I think it's some engine, there's other games that do that too (basically the whole point of the `gstate_c.uv` optimization.)  Makes sense to add them here.

-[Unknown]